.drone.yml

```diff
@@ -5461,6 +5460,168 @@ depends_on:
   - corr/real-time-environment-cyclictest-latencies
 
 
+---
+##############################################################################################################
+# This pipeline bootstraps a tezedge node to 1000 and measures memory allocation peaks during that
+##############################################################################################################
+kind: pipeline
+name: perf/memory-allocation-bootstrap-009-florence
+
+depends_on:
+  - build-tezedge-binaries
+
+trigger:
+  branches:
+    - master
+    - develop
+
+volumes:
+  - name: identities
+    host:
+      path: /usr/local/etc/tezedge-ci/build/build_${DRONE_BUILD_NUMBER}/build_files/identities
+  - name: snapshots
+    host:
+      path: /usr/local/etc/tezedge-ci/data/
+  - name: tools
+    host:
+      path: /usr/local/etc/tezedge-ci/tools
+
+steps:
+
+  - name: octez-node
+    image: tezos/tezos:v9-release
+    detach: true
+    environment:
+      TEZOS_NODE_DIR: /home/tezos/data
+    volumes:
+      - name: identities
+        path: /identities
+      - name: snapshots
+        path: /snapshots
+    commands:
+      - tezos-node config init --network florencenet
+      - cp /identities/identity_1.json $${TEZOS_NODE_DIR}/identity.json
+      - tezos-node snapshot import /snapshots/octez-009-florence-1000.snapshot --block BKqNZHy5CFYLD6v6BjDLrEavrkK4qH39hhrsD7QtbWcdnnwoYub
+      - tezos-node run --no-bootstrap-peers
+
+  - name: tezedge-new-node
+    image: tezedge/tezedge-ci-builder
+    user: root
+    volumes:
+      - name: identities
+        path: /identities
+      - name: tools
+        path: /tools
+    commands:
+      - OCTEZ_IP=$(perl -e 'use Socket; my $a = gethostbyname("octez-node"); print inet_ntoa($a)')
+      - echo $${OCTEZ_IP}
+      # use alloc instead of jemalloc
+      - awk '/allocator/ { print "//" $0; next } /.*/ { print }' light_node/src/main.rs > /tmp/main.rs && mv /tmp/main.rs light_node/src/main.rs
+      - cargo build --package light-node --package protocol-runner
+      - export LD_LIBRARY_PATH=tezos/sys/lib_tezos/artifacts/
+      - >
+        valgrind --tool=massif --massif-out-file=new-node.massif.out
+        target/debug/light-node --protocol-runner target/debug/protocol-runner
+        --network florencenet --peers $${OCTEZ_IP}:9732
+        --config-file light_node/etc/tezedge/tezedge_drone.config
+        --identity-file /identities/identity_2.json
+        --peer-thresh-low 1 --peer-thresh-high 1
+        --synchronization-thresh=0
```

Comment:
@akoptelov 
I would add here at least `--rpc-port 18732`

Reply:
> @akoptelov
> I would add here at least `--rpc-port 18732`

It is there already, in `light_node/etc/tezedge/tezedge_drone.config`. But to be more explicit here, yes, we can do that.